statefun-flink/statefun-flink-core/src/main/protobuf/http-function.proto

```diff
@@ -105,11 +105,34 @@ message FromFunction {
         google.protobuf.Any argument = 2;
     }
 
+    // DelayedInvocation represents a delayed remote function call with a target address, an argument
+    // and a delay in milliseconds, after which this message to be sent.
+    message DelayedInvocation {
+        // the amount of milliseconds to wait before sending this message
+        int64 delay_in_ms = 1;
+        // the target address to send this message to
+        Address target = 2;
+        // the invocation argument
+        google.protobuf.Any argument = 3;
+    }
+
+    // EgressMessage an argument to forward to an egress identified with an egress_identifier.
+    // an ingress identifier corresponds to the Java SDK class EgressIdentifier(namespace, name, consumedType)
+    // where the consumed type is a google.protobuf.Any and the namespace and name are derrived
+    // from the egress_identifier.
+    message EgressMessage {
+        // The target egress id in the form of <namespace>/<name>
+        string egress_identifier = 1;
```

Comment:
Should we be consistent with the `Address` message, where the `namespace` and `name` are 2 separate fields?

Reply:
Either that, or we change the `Address` message to use the Python / YAML way of formatting namespace + name pairs: `<namespace>/<name>` and do the parsing in the `RequestReplyFunction`